astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time`
+            Observation time.
+
+        target : `~astroplan.FixedTarget` or `~astropy.coordinates.SkyCoord`
+            Celestial object of interest.
+
+        Returns
+        -------
+        `~astropy.coordinates.Angle`
+            Parallactic angle
+        '''
+
+        if not (hasattr(target, 'ra') or hasattr(target, 'dec') or
+                hasattr(target, 'coord')):
+            raise TypeError('The target must be a coordinate (i.e. a '
+                            'FixedTarget or SkyCoord).')
+
+        if hasattr(target, 'coord'):
+            coordinate = target.coord
+        else:
+            coordinate = target
+
+        # Eqn (14.1) of Meeus' Astronomical Algorithms
+
+        H = (time.sidereal_time('mean', longitude=self.location.longitude)
```

Comment:
Can you split this out in a separate line?
```python
t = time.sidereal_time('mean', longitude=self.location.longitude)
```
I think it's more readable.
(if you disagree, no need to argue, OK to keep as is)